text/0005-global-init.md

```diff
@@ -51,10 +65,12 @@ party libraries to use the global SDK before it is installed, which is
 addressed in a requirement stated below.
 
 The explicit initializer method should take independent `Tracer` and
-`Meter` objects (e.g., `opentelemetry.Init(Tracer, Meter)`).  The SDK
-may be installed no more than once.  After the first SDK installed,
-subsequent calls to the explicit initializer shall log console
-warnings.
+`Meter` factories (e.g., `opentelemetry.Init(TracerFactory,
+MeterFactory)`), factories because they construct explicitly named
```

Comment:
Seems like there would be benefit to split this into two calls (being able to initialize the tracer and meter independently).  Any particular reason to keep these tied together in the same api call?

Reply:
I don't feel very strongly about this. I was imagining that it's important for an application to know that their metrics and traces began coming through at the same moment in the process lifetime.